src/internal/components/tooltip/index.tsx

```diff
@@ -39,7 +38,7 @@ export default function Tooltip({
 
   return (
     <Portal>
-      <div className={clsx(styles.root, className)} {...contentAttributes} data-testid={trackKey}>
+      <div className={styles.root} {...contentAttributes} data-testid={trackKey}>
```

Comment:
Why is `styles.root` staying here? it is also a test-util only style, what is the use-case of selecting it, if it is incorrectly placed in the DOM?

Why does this DOM node needed in the first place?

--

There is an avatar test util pointing on this DOM node too: https://github.com/cloudscape-design/chat-components/blob/81f675f568a82891661839f382262b80ea276289/src/test-utils/dom/avatar/index.ts#L14

Do you plan updating it in a separate PR?

Reply:
That's a good call out. I will create follow-up PRs that will 1) Remove the dependency on the tooltip wrapper from Avatar and 2) Remove the root style from this node.

I will also check if there is an opportunity to remove the node altogether. I think the "contentAttributes" and the testid can be passed down to the internal container.